windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -13,7 +13,10 @@
 #  ARGUMENTS
 ##############################
 
-Param([string]$version = (Get-Content "$PSScriptRoot\..\windows-installer\VERSION"))
+Param(
+  [string]$version = (Get-Content "$PSScriptRoot\..\windows-installer\VERSION"),
+  [string]$localOutputGemDir
```

Comment:
Where is this variable being set? From the build / test / release infra?

Reply:
Yes, I'm making the corresponding changes to the build scripts. This flag is optional, so it'll be ignored if unset.